systems/plants/RigidBodyFlatTerrain.m

```diff
@@ -3,13 +3,18 @@
 %  everywhere
   
   methods 
-    function obj = RigidBodyFlatTerrain()
+    function obj = RigidBodyFlatTerrain(varargin)
+      % Construct a flat terrain map at a given z height (default 0)
+      p = inputParser();
```

Comment:
ok, cool.  i didn't realize that.  
what about performance?  it seems a lot heavier than our minimal parsing of a structure (but perhaps the mathworks might go to greater effort to optimize this case).  would we want to use it, or avoid it, in e.g. forwardKin?